packages/core/src/components/legend-item/legend-item.tsx

```diff
@@ -0,0 +1,123 @@
+import { Component, ComponentInterface, Element, Event, EventEmitter, h, Host, Prop, State } from "@stencil/core";
+import clsx from "clsx";
+
+/**
+ * @slot - The label for this legend item. Omit the label when a `dso-selectable` is placed in slot `selectable`.
+ * The label should than be on the `dso-selectable`
```

Comment:
In deze JSDocs moeten we niet zelf aan regelafbreking doen, mijn IDE (vscode) snapt het dan niet meer:

❌ 
![image](https://github.com/dso-toolkit/dso-toolkit/assets/9696261/de467f8f-8f35-46ae-bb8a-b986161c580b)

✅ 
![image](https://github.com/dso-toolkit/dso-toolkit/assets/9696261/c5cd584a-5ade-42ee-b6ed-246777df09ad)

Reply:
✅